configure.ac

```
@@ -480,8 +492,11 @@ if test -z "$CXX"; then
 		x86_64-apple-tvos*-simulator)  CXX=x86_64-apple-tvos-simulator-clang++ ;;
 
 		aarch64-apple-watchos*-simulator) CXX=arm64-apple-watchos-simulator-clang++ ;;
-		aarch64-apple-watchos*)           CXX=arm64_32-apple-watchos-clang++ ;;
+		aarch64-apple-watchos*)           CXX=arm64-apple-watchos-clang++ ;;
```

This one was correct.
```suggestion
		aarch64-apple-watchos*)           CXX=arm64_32-apple-watchos-clang++ ;;
```

---

Oops. Thought I was correcting a typo for visionOS